pilot/pkg/security/authn/v1beta1/policy_applier_test.go

```diff
@@ -705,6 +704,15 @@ func TestJwtFilter(t *testing.T) {
 	for _, c := range cases {
 		t.Run(c.name, func(t *testing.T) {
 			istiotest.SetBoolForTest(t, &features.EnableRemoteJwks, c.enableRemoteJwks)
+			NewPolicyApplier("root-namespace", c.in, nil, push).JwtFilter()
+			for _, config := range c.in {
+				for _, jwtRule := range config.Spec.(*v1beta1.RequestAuthentication).JwtRules {
+					retryToCheckCache(5, 20*time.Millisecond, func() bool {
+						return push.JwtKeyResolver.CheckPubKeyExistInCache(jwtRule.Issuer, jwtRule.JwksUri)
+					})
+				}
+			}
+
 			if got := NewPolicyApplier("root-namespace", c.in, nil, push).JwtFilter(); !reflect.DeepEqual(c.expected, got) {
```

Comment:
should we use the `JwtFilter` from L707? Or if we are testing its not changed, then check both times?